### PR TITLE
Lightning Talks 2024 Slides Update

### PR DESCRIPTION
**Pull request by Bikram-ghuku**

### CODE REVIEW

File: 2024/Autumn/Lightning-Talks/Slides/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Lightning Talks 2023
```
Comment (harshit-jain52) [inline 2024/Autumn/Lightning-Talks/Slides/README.md:1]: It's 2024

**Review by SdSadat — COMMENTED** [review]
@Himesh-Kundal Change the commit message to something descriptive of your action.
something like: "slides link changed from canva to github."

File: 2023/Autumn/Lightning-Talks/Slides/README.md
Comment (harshit-jain52) [inline 2023/Autumn/Lightning-Talks/Slides/README.md:1]: not here :facepalm: 
2023 in 2023/
2024 in 2024/

**Comment by Bikram-ghuku** [comment]
Force push?

**Comment by Himesh-Kundal** [comment]
had to amend commit by rebasing

**Review by SdSadat — APPROVED** [review]
LGTM

**Review by harshit-jain52 — APPROVED** [review]

**Pull request merged**